m4_sessions/automation/session_one/five_noah_eric_nicke_laszlo.md

```diff
@@ -1,3 +1,51 @@
-## Group Five's Session Plan
+# Turing Career Development: Automation
 
-M4 Student Leaders: Noah, Eric, Nick E, Laszlo
+## Session Outcomes
+
+*  [Add detail here]
+
+## Pre-Session
+
+
+### Survey Questions
+
+Survey Questions to share with instructional staff by Wednesday of the same week.
+
+1.   What value would you assign to this session (from 1-5)?
+2.   What are you taking away from this session?
+3.   What role, if any, does Turing play in taking responsibility for this issue?
+
+
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
```

Comment:
who will send out the pre-reading? is the expectation that students will read/watch every resource you've listed?